docs/tools/apis/indexer-api.md

```diff
@@ -5,60 +5,142 @@ sidebar_position: 2
+import TabItem from '@theme/TabItem';
+import MainnetIcon from '@site/static/img/icons/lukso-signet-fuschia.svg';
+import TestnetIcon from '@site/static/img/icons/lukso-signet-yellow.svg';
+import QueryExamples from '@site/src/components/QueryExamples';
 
-This feature is currently in private beta. If you are interested, please [fill in this form](https://forms.gle/rhWA25m3jjuPNPva9).
+The [**Envio Indexer**](https://envio.lukso-mainnet.universal.tech/) provides a powerful way to query any information, past events or metadata from the LUKSO blockchain. It offers flexible querying capabilities with support for filtering and real-time subscriptions.
 
-:::
+![Envio Indexer](/img/tools/envio_indexer.png)
+
+## Get Started
+
+This guide covers how to query the LUKSO Indexer using GraphQL, with practical examples and code samples.
 
-If you are building on LUKSO, you can request a developer access to our indexer. This will allow you to query Universal Profiles and LSP7/8 assets.
+Choose the client that best fits your use case:
 
-## [Algolia](https://www.algolia.com/)
+- **[HTTP client](#http-client)**: Best for one-time queries, searches, and fetching data on demand
+- **[WebSocket client](#websocket-client)**: Best for real-time updates, monitoring changes, and live data feeds
 
-<p align="center">
-<img src="/img/tools/algolia_example.png" height="600px" alt="example of Algolia search"/>
-</p>
+:::success Interactive Playground
 
-Our backend system indexes profiles and assets and sends the data to Algolia.
+Explore the full schema and test queries in the interactive playground:
+**[https://envio.lukso-mainnet.universal.tech/](https://envio.lukso-mainnet.universal.tech/)**
+
+:::
 
-You can use the following Algolia API key to build a quick search engine for Universal Profiles and/or assets (as seen on the screenshot above). This is the technology that powers the search bar on [https://universalprofile.cloud/](https://universalprofile.cloud/).
+**GraphQL Endpoint:**
+
+<Tabs>
+  <TabItem value="mainnet" label={<><MainnetIcon style={{width: '20px', height: '20px', marginRight: '8px', marginTop: '4px', verticalAlign: 'text-bottom'}} /> Mainnet</>} default>
 
 ```
-APPLICATION_ID = YHFN1WRCR5
-API_KEY = [request it from us]
+https://envio.lukso-mainnet.universal.tech/v1/graphql
 ```
 
-**Indices:**
+  </TabItem>
+  <TabItem value="testnet" label={<><TestnetIcon style={{width: '20px', height: '20px', marginRight: '8px', marginTop: '4px', verticalAlign: 'text-bottom'}} /> Testnet</>}>
 
-- `prod_mainnet_assets`
-- `prod_testnet_assets`
-- `prod_mainnet_universal_profiles`
-- `prod_testnet_universal_profiles`
+```
+https://envio.lukso-testnet.universal.tech/v1/graphql
+```
 
-**Access Control Lists (ACLs):**
+  </TabItem>
+</Tabs>
 
-- `search`
+## HTTP Client
 
-### Resources
+Install the `graphql-request` library to interact with the indexer:
 
-- [Algolia Docs](https://www.algolia.com/doc/)
+```bash
+npm install graphql-request graphql
```

Comment:
There are other libraries, maybe say "any graphql client library" will work, this is an example.